app/models/metric/purging.rb

```diff
@@ -69,7 +69,7 @@ def self.purge_hourly(older_than, window = nil, total_limit = nil, &block)
     purge_by_date(older_than, "hourly", window, total_limit, &block)
   end
 
-  def self.purge_realtime(older_than, window = nil, total_limit = nil, &block)
+  def self.purge_realtime(older_than, _window = nil, _total_limit = nil)
```

Comment:
Some of these act like documentation for subclassess knowing what to pass - that being said, I'm fine with this.

Reply:
Are subclasses supposed to be redefining methods if the core class already has an implementation, though? This one calls `truncate_child_tables(older_than)`. Is the expectation that subclasses will call `super + custom_implementation`?

Reply:
subclasses are always free to override, and yes, they generally use `super + custom`.  However, I'd agree that it's unlikely they'll override this particular method; it's usually the abstract methods defined with `NotImplementedError` that they override.